test/format/config_def.c

```diff
@@ -0,0 +1,349 @@
+/* DO NOT EDIT: automatically built by format/config.sh. */
+
+#include "format.h"
+
+CONFIG configuration_list[] = {
```

Comment:
Sounds like a recent change has renamed the file from `config_def.c` to `format_config_def.c`. Let's remove config_def.c from the PR